tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
add tslint to the project. that will help you in the future

Reply:
done